.circleci/config.yml

```diff
@@ -1,20 +1,31 @@
 version: 2.1
 
-jobs:
-  pr-tests:
+executors:
+  executor:
     machine:
       image: ubuntu-2004:current
     resource_class: large
+
+jobs:
+  build:
+    executor: executor
     steps:
+      - checkout
```

Comment:
how can it do checkout without git? or do we assume the image has git?

Reply:
It's a circleci-specific thing.